src/editorialreview/logic.py

```diff
@@ -84,18 +84,21 @@ def handle_generated_form_post(review_assignment, request):
 
     for field in file_fields:
         if field.element.name in request.FILES:
-            # TODO change value from string to list [value, value_type]
             save_dict[field.element.name] = [
                 handle_review_file(
                     request.FILES[field.element.name],
-                    review_assignment, 'reviewer')
+                    review_assignment,
+                    'reviewer'
+                )
             ]
 
     for field in data_fields:
-        if field.element.name in request.POST:
-            # TODO change value from string to list [value, value_type]
-            save_dict[field.element.name] = [
-                request.POST.get(field.element.name), 'text']
+        field_name = field.element.name.encode('ascii', 'xmlcharrefreplace')
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?